### DISCUSSION TITLE
How to set up two type solids in reactiveTransportDBS？

### DISCUSSION THREAD

**Discussion by Kolihanar** [GeoChemFoam/GeoChemFoam#46] [Q&A] [answered]
Hi, I wonder if this solver can be set up with the same acidic fluid flowing in and then set up with two different solids being dissolved?Thaks！

**Comment by GeoChemFoam** [comment, marked as answer]
The solver doesn’t currently support this so you will need to learn some open foam and edit the source code.


On Mar 27, 2024, at 10:31, Kolihanar ***@***.***> wrote:

﻿
****************************************************************
Caution: This email originated from a sender outside Heriot-Watt University.
Do not follow links or open attachments if you doubt the authenticity of the sender or the content.
****************************************************************


Hi, I wonder if this solver can be set up with the same acidic fluid flowing in and then set up with two different solids being dissolved?Thaks！

—
Reply to this email directly, view it on GitHub<https://github.com/GeoChemFoam/GeoChemFoam/discussions/46>, or unsubscribe<https://github.com/notifications/unsubscribe-auth/AWDOAFVB3FHHKCBCJS2XMUTY2KNZDAVCNFSM6AAAAABFKTA5QCVHI2DSMVQWIX3LMV43ERDJONRXK43TNFXW4OZWGQZDOMBRHA>.
You are receiving this because you are subscribed to this thread.Message ID: ***@***.***>

________________________________

Founded in 1821, Heriot-Watt is a leader in ideas and solutions. With campuses and students across the entire globe we span the world, delivering innovation and educational excellence in business, engineering, design and the physical, social and life sciences. This email is generated from the Heriot-Watt University Group, which includes:

  1.  Heriot-Watt University, a Scottish charity registered under number SC000278
  2.  Heriot- Watt Services Limited (Oriam), Scotland's national performance centre for sport. Heriot-Watt Services Limited is a private limited company registered is Scotland with registered number SC271030 and registered office at Research & Enterprise Services Heriot-Watt University, Riccarton, Edinburgh, EH14 4AS.

The contents (including any attachments) are confidential. If you are not the intended recipient of this e-mail, any disclosure, copying, distribution or use of its contents is strictly prohibited, and you should please notify the sender immediately and then delete it (including any attachments) from your system.
